lib/screens/profile_screens/recovery_phrase/interactor/usecases/generate_recovery_words_use_case.dart

```diff
@@ -0,0 +1,17 @@
+// ignore: import_of_legacy_library_into_null_safe
+import 'package:eosdart_ecc/eosdart_ecc.dart';
+import 'package:seeds/datasource/local/settings_storage.dart';
+import 'package:seeds/utils/mnemonic_code/hex.dart';
+import 'package:seeds/utils/mnemonic_code/mnemonic_code.dart';
+
+/// Uses the private key to generate 16 words recovery phrase
+class GenerateRecoveryPhraseUseCase {
+  List<String> run() {
+    // Construct the EOS private key from string
+    final EOSPrivateKey privateKey = EOSPrivateKey.fromString(settingsStorage.privateKey);
+    final String entropy = HEX.encode(privateKey.d);
+    final String mnemonic = entropyToMnemonic(entropy);
+
+    return mnemonic.split('-');
```

Comment:
This actually returns 24 words is that rigth?
![image](https://user-images.githubusercontent.com/42857405/135174660-b52a27ed-0aee-466c-9d3f-3e6d0553c4cb.png)

Reply:
you are 100% correct. 
I need to figure out a way to generate less words....

Reply:
any idea?

Reply:
Maybe a shorter `entropyString` the current generated seems have a lenght of 32.
![image](https://user-images.githubusercontent.com/42857405/135180816-e7bd471d-1403-41f7-872b-635286daa06c.png)